backend/director/core/session.py

```
@@ -363,3 +363,19 @@ def get_all(self):
+        valid_events = {
+            "collection_update": "collections",
+        }
+
+        event_name = valid_events.get(event_type)
+        if not event_name or not isinstance(data, dict) or not data:
+            return
+
+        try:
+            emit(event_name, data, namespace=namespace)
+        except Exception:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)